### PR TITLE
java: modify Java example to move healthService to the admin server

### PR DESCRIPTION
**Pull request by sanjaypujare**
No need to have both health and admin server. So we just add health service to the admin server

CC @ericgribkoff @yashykt @easwars @lidizheng 

As per the discussion

### CODE REVIEW

**Comment by ericgribkoff** [comment]
Doesn't our documentation describe creating health checks using the gRPC protocol and the serving port? It looks like moving the health server to a non-serving port would break e.g. https://cloud.google.com/traffic-director/docs/set-up-proxyless-gce#creating_the_health_check_firewall_rule_and_backend_service without an accompanying update to the documentation

**Comment by sanjaypujare** [comment]
> Doesn't our documentation describe creating health checks using the gRPC protocol and the serving port? It looks like moving the health server to a non-serving port would break e.g. https://cloud.google.com/traffic-director/docs/set-up-proxyless-gce#creating_the_health_check_firewall_rule_and_backend_service without an accompanying update to the documentation

Eric, nothing will break. There is also (old code, check https://github.com/GoogleCloudPlatform/traffic-director-grpc-examples/blob/master/java/src/main/java/io/grpc/examples/wallet/StatsServer.java#L179) a health-service on the serving port which is not being removed. The only change is to not create a separate plaintext health-check only port but to overload the admin port to also be the plaintext health check port.

**Comment by sanjaypujare** [comment]
Friendly ping. Any comments? Can this be approved?

**Review by ericgribkoff — APPROVED** [review]

File: java/src/main/java/io/grpc/examples/wallet/AccountServer.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
 import io.grpc.examples.wallet.account.MembershipType;
 import io.grpc.health.v1.HealthCheckResponse.ServingStatus;
 import io.grpc.protobuf.services.ProtoReflectionService;
+import io.grpc.services.AdminInterface;
```
[the remark this answers is not in the document]
Comment (sanjaypujare) [reply java/src/main/java/io/grpc/examples/wallet/AccountServer.java:33]: So what date would that be? Do we also need to update the docs to reference 1.39?

[the remark this answers is not in the document]
Comment (sanjaypujare) [reply java/src/main/java/io/grpc/examples/wallet/AccountServer.java:33]: >  I think it's okay to add admin service back to the Account server.

That just applies to the Go wallet example. Isn't that right? The Java example changes are not blocked by Go issues

[the remark this answers is not in the document]
Comment (lidizheng) [reply java/src/main/java/io/grpc/examples/wallet/AccountServer.java:33]: That would be June 29th for v1.39.0 release. I was hope Go team would provide some advice here. Let's merge it to master, the user guides are using `v1.0.x` branch, so we should be fine.

**Review by lidizheng — APPROVED** [review]

**Pull request merged**